Values and Q1 Goals for Words & Pictures (#10067)
PostHog/posthog.com: contents/teams/words-pictures/index.mdx

---
title: Words & Pictures
template: team
---

## Values

### 1. Progress is more important than process

We can be a chaotic team, but that's OK because we're also good at getting stuff done. We'll create process when it's helpful, but if it's a choice between doing something good/fast and doing something "properly", we'll choose good/fast.

### 2. Sparking joy is a multidisciplinary job

We're called Words & Pictures, but that's not the whole story. We think in animations, emails, events, songs, videos, vibes, and more. There's no such thing as "just a writer" or "just a designer" here. 
  
### 3. We're not an 'ideas' team

> You know the type of 'Ideas' person, who comes up with big ideas and then doesn't do anything with it, or tries to get someone else to do it for them? Yeah, we hate people like that. We ship.

Author: ```suggestion
You know the type of 'Ideas' person who comes up with big ideas but doesn't do anything with them, or tries to get someone else to do it? Yeah, we hate people like that. We ship.
```

---
title: Words & Pictures
template: team
---

## Values

### 1. Progress is always more important than process

We can be a chaotic team, but that's OK because we're also good at getting stuff done. We'll create process when it's helpful, but if it's a choice between doing something good/fast and doing something "properly", we'll choose good/fast. We encourage each other not to ask for permission on things. 

### 2. Sparking joy is a multidisciplinary, inclusive job

> We're called Words & Pictures, but that's not all we do. We think in animations, emails, events, songs, videos, vibes, and more. There's no such thing as "just a writer" or "just a designer", and everyone's feedback is valued equally on all projects.

Reviewer: 'We think in....' etc. is a bit twee for my liking. I think this point would be stronger if it's more around continuing to push the brand further - let's keep being much more fun, weird and joyful than every other generic software company out there.

  
### 3. We're not an 'ideas' team

You know the type of 'Ideas' person who comes up with big ideas but doesn't do anything with them, or tries to get someone else to do it? Yeah, we hate people like that too. We ship things and, wherever possible, we ship them all by ourselves. 

## Responsibilities

- Merch and all related creative work
- Customer-facing communications
- Emails, ads, and other non-content marketing
- Special programs, such as PostHog for Startups

> - Partnerships, PR, product launches

Reviewer: ```suggestion
- Product launches
```

- Events, when they happen

> ## How we work

Reviewer: Maybe add a link to the budget here as well?
